R/calculate_annual_pd_changes.R

```diff
@@ -36,6 +41,15 @@ calculate_pd_change_annual <- function(data,
     )
   )
 
+  validate_data_has_expected_cols(
```

Comment:
since it is duplicate between annual and overall please move this, and the distinct below to a separate function and apply it to exposure_at_default before passing it to the function